src/docker/Dockerfile

```diff
@@ -119,6 +120,16 @@ RUN wget -O /usr/local/bin/gomplate "https://github.com/hairyhenderson/gomplate/
   && chmod +x /usr/local/bin/gomplate \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+ # Install Podman. Get versions from https://github.com/hairyhenderson/gomplate/releases
```

Comment:
```suggestion
# Install Podman. Get versions from https://github.com/hairyhenderson/gomplate/releases
```

Reply:
will handle this.

Reply:
I will update this readme, based on the new version podman.